.ci/aarch64/clean_up_aarch64.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+stale_process_union=( "docker-containerd-shim" "qemu-system-aarch64" )
+stale_docker_mount_point_union=( "/var/lib/docker/containers" "/var/lib/docker/overlay2" )
+stale_docker_dir="/var/lib/docker"
+stale_kata_dir="/var/lib/vc/sbs"
+
+lib_script="${GOPATH}/src/${tests_repo}/.ci/lib.sh"
+source "${lib_script}"
+
+kill_stale_process()
+{
+	for stale_process in "${stale_process_union[@]}"; do
+		local pid_union=$(ps -eo pid,command | grep "${stale_process}" |  grep -v "grep" | awk '{print $1}')
```

Comment:
you might be able to use `pkill` here and do away with much of this logic. Also see `pgrep`